src/services/db/GitFileSystemService.ts

```diff
@@ -50,6 +50,21 @@ export default class GitFileSystemService {
     this.git = git
   }
 
+  hasGitFileLock(repoName: string): ResultAsync<boolean, GitFileSystemError> {
+    const gitFileLockPath = ".git/index.lock"
+    return this.getFilePathStats(repoName, gitFileLockPath)
```

Comment:
will this check for repoName/.git/index.lock -> check if `/` is added automatically

Reply:
Yup, `getFilePathStats` checks for `${EFS_VOL_PATH}/${repoName}/${filePath}`